### PR TITLE
pitr: support restart log-backup task and remove several know issues.

### PR DESCRIPTION
**Pull request by joccau**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by 3pointer — APPROVED** [review]

File: br/pitr-known-issues.md [hunk 1]
```diff
@@ -61,12 +50,5 @@ Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
...
 
-## 在 GCS 或 Azure Blob Storage 上第一次执行 `PITR Truncate` 命令时报错
-
-Issue 链接：[#38229](https://github.com/pingcap/tidb/issues/38229)
-
-在 GCS 或 Azure Blob Storage 上**第一次**执行 `PITR Truncate` 时会提示文件 `v1_stream_trancate_safepoint.txt` 不存在。解决方法如下：
-
-在 PITR 的备份根目录下，创建文件 `v1_stream_trancate_safepoint.txt`，然后写入 `0`。注意文件不得包含其他字符。在第一次执行 `PITR Truncate` 命令时，添加该文件即可。
 <!-- TODO: v6.4.0 发版时，取消注释以下文字  -->
 <!-- 或者，使用 v6.4.0 及以上版本的 BR。 -->
```
Comment (Leavrth) [inline br/pitr-known-issues.md:54]: 这两行注释也属于这个已知问题里面的，也可以删除

**Review by Leavrth — APPROVED** [review]

File: br/pitr-known-issues.md (hunk 1 above)
Comment (IANTHEREAL) [reply br/pitr-known-issues.md:54]: Rest LGTM

File: br/br-log-command-line.md [hunk 2]
```diff
@@ -231,6 +231,8 @@ Global Flags:
 
 ### （永久）停止日志备份任务
 
+#### 停止日志备份任务
```
Comment (shichun-0415) [inline br/br-log-command-line.md:234]: 上面的三级标题改为“停止和重启日志备份任务”？

File: br/br-log-command-line.md [hunk 3]
```diff
@@ -255,15 +257,22 @@ Global Flags:
 
 > **警告：**
 >
-> - 请谨慎使用该命令。只有在你确认不再继续使用 PiTR 的情况下，才可以停止日志备份任务。如果你只需暂停日志备份，请使用 `br log pause` 和 `br log resume` 命令。
-> - 如果你选择使用 `br log stop` 停止备份任务，在使用 `br log start` 重启备份任务时，需要指定一个与之前不同的日志备份存储路径，而不同的日志备份路径会导致你无法使用 `br restore point` 进行一键恢复。
+> - 请谨慎使用该命令，如果你只需**暂时停止**日志备份，请使用 `br log pause` 和 `br log resume` 命令。
```
Comment (shichun-0415) [inline br/br-log-command-line.md:260]:
```suggestion
> 请谨慎使用该命令，如果你只需**暂时停止**日志备份，请使用 `br log pause` 和 `br log resume` 命令。
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

File: br/br-log-command-line.md [hunk 4]
```diff
@@ -229,9 +229,11 @@ Global Flags:
 ./br log resume --task-name=pitr --pd=172.16.102.95:2379
 ```
 
-### （永久）停止日志备份任务
+### 停止和重启日志备份任务
```
Comment (qiancai) [inline br/br-log-command-line.md:233]: 这里是否可以加一句话介绍下这个小节

File: br/br-log-command-line.md [hunk 5]
```diff
@@ -253,17 +255,24 @@ Global Flags:
...
 
 使用示例：
 
 ```shell
 ./br log stop --task-name=pitr --pd=172.16.102.95:2379
 ```
 
+#### 重新启动备份任务
+
+当使用 `br log stop` 命令停止日志备份任务后，可在另一个 `--storage` 路径下重新创建一个新的日志备份任务，也可以在原来的 `--storage` 路径下重新启动日志备份任务。如果是续接上一次的任务，此时需要注意：
```
Comment (qiancai) [inline br/br-log-command-line.md:270]:
```suggestion
当使用 `br log stop` 命令停止日志备份任务后，可在另一个 `--storage` 路径下重新创建一个新的日志备份任务，也可以在原来的 `--storage` 路径下执行 `br log start` 命令重新启动日志备份任务。如果是在原来的 `--storage` 路径重启任务，需要注意：
```

File: br/pitr-known-issues.md [hunk 6]
```diff
@@ -60,13 +49,3 @@ Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
 - 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性。
 - 如果先启动添加索引加速任务，再创建 PITR 备份任务，此时 PITR 备份任务会报错，但不影响正在添加索引的任务。
 - 如果同时启动 PITR 备份任务和添加索引加速任务，可能会由于两个任务无法察觉到对方而导致 PITR 不能成功备份增加的索引数据。
-
-## 在 GCS 或 Azure Blob Storage 上第一次执行 `PITR Truncate` 命令时报错
-
```
Comment (qiancai) [inline br/pitr-known-issues.md:65]: [https://github.com/pingcap/docs-cn/pull/11836/files](https://github.com/pingcap/docs-cn/pull/11836/files) 修改了这个报错的解决办法，到底是删还是改呢。

File: br/br-log-command-line.md [hunk 7]
```diff
@@ -229,9 +229,11 @@ Global Flags:
 ./br log resume --task-name=pitr --pd=172.16.102.95:2379
 ```
 
-### （永久）停止日志备份任务
+### 停止和重启日志备份任务
```
Comment (shichun-0415) [inline br/br-log-command-line.md:232]:
```suggestion
### 停止和重启日志备份任务

通过执行 `br log stop` 命令，你可以停止正在进行的日志备份任务。停止的任务，可以通过 `--storage` 路径重新启动。
```

File: br/br-log-command-line.md (hunk 4 above)
Comment (shichun-0415) [reply br/br-log-command-line.md:233]: Added in [f8cb7a4](https://github.com/pingcap/docs-cn/pull/11832/commits/f8cb7a426a418a6fdae3c5a8e226c661c0470869)

**Review by YuJuncen — APPROVED** [review]

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**